packages/design-system/src/styles/components/_Choice.scss

```diff
@@ -29,6 +29,18 @@ $ds-c-inset-border-width: $spacer-half;
   }
 }
 
+// Style radio and checkboxes when errored
+.ds-c-fieldset--error {
+  .ds-c-choice + label::before,
+  .ds-c-choice:checked + label::before {
+    border-color: $color-error;
+  }
+  .ds-c-choice--inverse + label::before,
+  .ds-c-choice--inverse:checked + label::before {
+    border-color: $color-error-light;
```

Comment:
This is great, I think this is what we should be doing, but I wanted to point out this isnt consistent with most of our form elements, for both HTML and React.

I.e. the html TextField example doesnt make the input border red.
<img width="528" alt="Screen Shot 2020-09-08 at 11 35 13 AM" src="https://user-images.githubusercontent.com/5424713/92503867-790e7d00-f1c7-11ea-854b-7249a066217b.png">

The React TextField applies an error border, but it doesnt apply a lighter error border with inverse.
<img width="546" alt="Screen Shot 2020-09-08 at 11 39 40 AM" src="https://user-images.githubusercontent.com/5424713/92504474-55980200-f1c8-11ea-8882-cfcee96c552b.png">

The way we implement inverse in these scenarios is also important, like in this case we rely on the parent element having `.ds-c-choice--inverse`, but in other components we directly have `ds-c-field--error` classes. We also have to consider how our implementation results in complexity for child design systems, like i dont think our child design systems are overriding or even aware of this `error-light` pattern.

Reply:
basically ^^ this was a long winded way of saying i think we really need to think about how we deal with error and inverse styles across our inputs in general.